open-lens/webpack/renderer.ts

```diff
@@ -117,79 +118,83 @@
     new CopyPlugin({
       patterns: [
         {
-          // TODO: Figure out a way to access without relative path
-          from: "../../node_modules/@k8slens/core/static/build/library/*.ttf",
+          from: path.resolve(
+            path.dirname(require.resolve("@k8slens/core/package.json")),
+            corePackageJson.exports["./fonts/*"]
+          ),
           to: "[name][ext]",
         },
       ],
     }),
 
-    ...(
-      isDevelopment
-        ? [new ReactRefreshWebpackPlugin()]
-        : []
-    ),
+    ...(isDevelopment ? [new ReactRefreshWebpackPlugin()] : []),
   ],
 };
 
- /**
-  * Import icons and image files.
-  * Read more about asset types: https://webpack.js.org/guides/asset-modules/
-  */
- export function iconsAndImagesWebpackRules(): webpack.RuleSetRule[] {
-   return [
-     {
-       test: /\.svg$/,
-       type: "asset/source", // exports the source code of the asset, so we get XML
-     },
-     {
-       test: /\.(jpg|png|ico)$/,
-       type: "asset/resource", // path to file, e.g. "/static/build/assets/*"
-     },
-   ];
- }
+/**
+ * Import icons and image files.
+ * Read more about asset types: https://webpack.js.org/guides/asset-modules/
+ */
+export function iconsAndImagesWebpackRules(): webpack.RuleSetRule[] {
+  return [
+    {
+      test: /\.svg$/,
+      type: "asset/source", // exports the source code of the asset, so we get XML
+    },
+    {
+      test: /\.(jpg|png|ico)$/,
+      type: "asset/resource",
+      generator: {
+        filename: "images/[name][ext]",
```

Comment:
```suggestion
        filename: "icons/[name][ext]",
```
? Or just keeping the svg separate? I don't really follow how they are accessed

Reply:
They are separate and of different assert style. see line 142 above. They are `assert/source` which means that the `import X from "<path-to-svg>"` get converted to `const X = "<some-string>";`